lms/djangoapps/discussion/static/discussion/js/discussion_router.js

```diff
@@ -18,7 +18,7 @@
                 initialize: function(options) {
                     Backbone.Router.prototype.initialize.call(this);
                     _.bindAll(this, 'allThreads', 'showThread');
-                    this.courseId = options.courseId;
+                    this.rootUrl = options.rootUrl;
```

Comment:
1. Yes, options are not passed down from view to view. Each view has its own options.
2. I'm going to do a quick sweep and see if I can get rid of Python naming.